5-WebApp-AuthZ/5-1-Roles/Infrastructure/AppRoles.cs

```diff
@@ -13,4 +13,10 @@ public static class AppRoles
         public const string UserReaders = "UserReaders";
         public const string DirectoryViewers = "DirectoryViewers";
     }
+
+    public static class AppPolicies
+    {
+        public const string UserReadersOnly = "UserReadersOnly";
```

Comment:
nit: not a huge fan of the names, but i also don't have a suggestion at the moment. :/ :)